src/db.rs

```diff
@@ -73,6 +76,20 @@ pub enum ExecuteCommandError {
 
     #[error(transparent)]
     SystemTimeError(#[from] SystemTimeError),
+
+    #[error("Internal state error")]
+    InternalStateError,
+}
+
+// Implementing Display is the easiest way to get a to_string() method
+impl Display for Evictor {
```

Comment:
We don't need to `impl Display`, it will cause un-necessry allocations everytime. I suggest that we have global static variables like this.

```rust
static EVICTOR_NOP: &[u8] = b"NOP";
static EVICTOR_RANDOM: &[u8] = b"RANDOM";
static EVICTOR_LRU: &[u8] = b"LRU";
```
and create the string frame like this
```rust
Frame::String(Bytes::from_static(EVICTOR_RANDOM));
```

Reply:
Sounds, good. We can add a similar method to get a `&[u8]`. Also I think it's better to name the methods as `as_str()` and `as_bytes()`.